src/databricks/labs/ucx/cli.py

```diff
@@ -611,6 +611,7 @@ def create_ucx_catalog(w: WorkspaceClient, prompts: Prompts, ctx: WorkspaceConte
     """
     workspace_context = ctx or WorkspaceContext(w)
     workspace_context.catalog_schema.create_ucx_catalog(prompts)
+    workspace_context.progress_tracking_installation.run()
```

Comment:
we should reflect this in the `assign_metastore`

Reply:
See this issue on this: https://github.com/databrickslabs/ucx/issues/2737